domains/kyizl.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "oqzx",
+    "email": "ekn@envs.net"
+  },
+  "record": {
+    "CNAME": "rev.kyizl.workers.dev"
+  },
```

site does not seem to be developer related
```suggestion
    "CNAME": "rev.kyizl.workers.dev"
  }
```